docs/modules/ROOT/pages/migration.adoc

```diff
@@ -0,0 +1,288 @@
+= Migration
+
+To migrate an existing LangChain application to RAGStack and use RAGStack's pinned, tested version of LangChain instead, you need to uninstall LangChain and install RAGStack.
+
+. `pip uninstall langchain`
+. `pip install ragstack-ai --upgrade-strategy="only-if-needed"`
+. `pip list` to see your current list of packages.
+
+An example migration is shown below.
+
+== RAGStack packages
+RAGStack contains the following packages as of version 0.3.1.
+You can replace any of these packages in your existing application with the `ragstack-ai` version.
```

Comment:
RAGstack doesn't contain these packages. They are transitive dependencies and their final versions will be resolved on the user environment.
I think we should only mention the dependency versions that we pin (langchain, llama-index, astrapy, cassio, ...)
Currently
```toml
[tool.poetry.dependencies]
python = ">=3.9,<3.12"
astrapy = "~0.6.2"
cassio = "~0.1.3"
unstructured = "^0.10"
llama-index = { version = "0.9.14", extras = ["langchain"] }
langchain = { version = "0.0.343", extras = ["openai"] }
langchain-core = "0.0.8"
```